.github/workflows/prerelease_windows_on_demand.yml

```diff
@@ -77,6 +77,12 @@ jobs:
         shell: pwsh
         run: build\windows\package_zip.ps1 -arch ${{ matrix.goarch }} -version ${{env.FAKE_TAG}}
 
+      - name: Generate checksum files
+        uses: ./src/github.com/newrelic/infrastructure-agent/.github/actions/generate-checksums
```

Comment:
Is this path ok? I see the previous actions referencing relative paths like `build/windows/...`

Reply:
Is the same one as
https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/prerelease_windows.yml#L79 
and
 https://github.com/newrelic/infrastructure-agent/blob/master/.github/workflows/component_windows_packaging.yml#L83

I guess you can use both paths when using local actions 🤷‍♂️